packages/config/src/VitessceAutoConfig.js

```diff
@@ -135,6 +159,14 @@ class AnndataZarrAutoConfig extends AbstractAutoConfig {
       });
     });
 
+    // if length of path is 1, storing the value as an array doesn't work
```

Comment:
Can you add to this comment with a concrete example, I am not sure I understand the issue

Reply:
The issue is that the cell sets don't get coloured. Here is a particular example: https://s3.amazonaws.com/vitessce-data/0.0.33/main/human-lymph-node-10x-visium/human_lymph_node_10x_visium.h5ad.zarr I am adding it to the comment.